test/test_functionality.py

```diff
@@ -112,6 +112,9 @@ def setUp(self):
         self.ctrl.load()
         self.tlm = self.ctrl.getTimeLayerManager()
 
+    def test_setup(self):
```

Comment:
this is what is making travis fail now.
The class TestForLayersWithOnePointPerSecond is abstract (as much as you can create abstract classes in Python with the help of the abc module, at least) so adding a test to it makes nosetests try to instatiate it, which doesn't work. I think we don't strictly need a test for self.qgs being not None, but if you want to add one, you can add it to the subclasses.